assets/xml/objects/object_im.xml

```diff
@@ -1,76 +1,79 @@
-        <Texture Name="object_im_Tex_006F40" OutName="tex_00006F40" Format="rgba16" Width="16" Height="16" Offset="0x6F40"/>
-        <Texture Name="object_im_TLUT_007140" OutName="tlut_00007140" Format="rgba16" Width="8" Height="13" Offset="0x7140"/>
-        <Texture Name="object_im_Tex_007210" OutName="tex_00007210" Format="ci8" Width="32" Height="32" Offset="0x7210"/>
-        <Texture Name="object_im_Tex_007610" OutName="tex_00007610" Format="ci8" Width="16" Height="16" Offset="0x7610"/>
-        <Texture Name="object_im_Tex_007710" OutName="tex_00007710" Format="ci8" Width="8" Height="8" Offset="0x7710"/>
+        <Texture Name="gImpaBrownGradientTex" OutName="brown_gradient" Format="ci8" Width="8" Height="8" Offset="0x1A80" TlutOffset="0x1880"/>
+        <Texture Name="gImpaSkinAndArmGuardTex" OutName="skin_and_armguard" Format="ci8" Width="16" Height="32" Offset="0x1AC0" TlutOffset="0x1880"/>
+        <Texture Name="gImpaElbowTex" OutName="elbow" Format="ci8" Width="16" Height="16" Offset="0x1CC0" TlutOffset="0x1880"/>
+        <Texture Name="gImpaShoulderPadBottomEdgeTex" OutName="shoulder_pad_bottom_edge" Format="ci8" Width="8" Height="8" Offset="0x1DC0" TlutOffset="0x1880"/>
+        <Texture Name="gImpaShoulderSkinTex" OutName="shoulder_skin" Format="ci8" Width="8" Height="16" Offset="0x1E00" TlutOffset="0x1880"/>
+        <Texture Name="gImpaChestPlateTex" OutName="chest_plate" Format="ci8" Width="8" Height="16" Offset="0x1E80" TlutOffset="0x1880"/>
+        <Texture Name="gImpaEmblemTex" OutName="emblem" Format="ci8" Width="16" Height="32" Offset="0x1F00" TlutOffset="0x1880"/>
+        <Texture Name="gImpaSpearEmblemTex" OutName="spear_emblem" Format="ci8" Width="16" Height="32" Offset="0x2100" TlutOffset="0x1880"/>
+        <Texture Name="gImpaChestPlateStrapTex" OutName="chest_plate_strap" Format="ci8" Width="16" Height="16" Offset="0x2300" TlutOffset="0x1880"/>
+        <Texture Name="gImpaChestPlateEdgeTex" OutName="chest_plate_edge" Format="ci8" Width="8" Height="16" Offset="0x2400" TlutOffset="0x1880"/>
+        <Texture Name="gImpaNeckTex" OutName="neck" Format="ci8" Width="32" Height="32" Offset="0x2480" TlutOffset="0x1880"/>
+        <Texture Name="gImpaSwordHandleTex" OutName="sword_handle" Format="rgba16" Width="8" Height="16" Offset="0x2880"/>
+        <Texture Name="gImpaPantsAndSkinTex" OutName="pants_and_skin" Format="rgba16" Width="16" Height="32" Offset="0x2980"/>
+        <Texture Name="gImpaBlackTex" OutName="black_square" Format="ci8" Width="8" Height="8" Offset="0x2D80" TlutOffset="0x1880"/> <!-- Used on the bottom of the leg -->
+        <DList Name="gImpaChestDL" Offset="0x5040"/>
+        <DList Name="gImpaRightShoulderDL" Offset="0x5D60"/>
+        <DList Name="gImpaRightArmDL" Offset="0x6048"/>
+        <DList Name="gImpaLeftShoulderDL" Offset="0x6200"/>
+        <DList Name="gImpaLeftArmDL" Offset="0x64E8"/>
+        <DList Name="gImpaTorsoDL" Offset="0x6720"/>
```

Comment:
I dont this type of name is used anywhere else, while `torso` is used in most other actors like this.

Reply:
I mean it's very much not her torso so

Reply:
just to make sure this is this limb, right

<details>

<summary>
click for z64utils screenshot with limb 2 (1-indexed) in red
</summary>

![](https://421.es/doyu/26dd2k)

</details>

Reply:
Just saw object_zo has gZoraPelvisLimb:

<details>

![](https://421.es/doyu/26yd88)

</details>

so maybe "pelvis" instead of "groin" if you like

(we would need anatomy guidelines for naming limbs...)